frontend/src/features/permits/pages/Application/components/review/ConfirmationCheckboxes.scss

```diff
@@ -6,10 +6,15 @@
   & &__checkbox {
     padding-left: 0;
     color: $bc-primary-blue;
-
+    
+    
     &--invalid {
       color: $bc-red;
     }
+    
+    &.Mui-disabled {
```

Comment:
In the context of the ApplicationInQueueReview.tsx component, these checkboxes are disabled, since the staff user reviewing the application should not be making/editing the attestations themselves.

Reply:
I'm wondering if it's possible to override the "disabled" className (ie. replace the MUI disabled className for that element with our custom className)

Reply:
We can add a modifier to the className based on the shouldDisableCheckboxes condition (e.g. confirmation-checkboxes__checkbox--disabled), however as with many other cases in the application, this is not specific enough to overwrite MUI's classNames and we usually have to fall back on including the MUI specific className in order to override the MUI styles correctly.

Reply:
I'm pretty sure there is a way to override the styles, we just need to be a bit creative (eg. using the rules with something like ".confirmation-checkboxes__checkbox.confirmation-checkboxes__checkbox--disabled"), you can play around with this a bit

Reply:
or use the "classes" prop to the component, and set the "disabled" property with a custom className and go from there